### PR TITLE
Proposed github process modification.

### PR DESCRIPTION
**Pull request by jsync-swirlds**
This proposed modification adds detail for how to handle specification text that is excessively large, and would only obscure the substance of a HIP if included inline.
* Clean up the README file, wrap long lines for readbility, and fix some typographic issues.
* Clean up the hip-0000-template.md file, wrap long lines for readability (and to encourage the same in HIP documents), and fix some typographic issues
* Added content to the "Specification" section to describe a recommended method for handling excessively large specification text, along with a small example.
* Added a folder and sample file (copied from HIP-904 implementation) to support the example.

### CODE REVIEW

**Review by Nana-EC — DISMISSED** [review]
LG. 1 suggestion and 1 Q

File: README.md [hunk 1]
```diff
@@ -4,66 +4,93 @@
...
 ## Submit a HIP
-
 1. Fork this repository
-2. Fill out this template: [hip template](hip-0000-template.md)
-3. Create a pull request against hashgraph/hedera-improvement-proposal main
-
-But what category do I make my HIP? See [hip-1 HIP types](HIP/hip-1.md#hip-types)
-
-See [hip-1](HIP/hip-1.md) for details on the HIP process or watch [this video](https://www.youtube.com/watch?v=Gbk8EbtibA0)
+1. Fill out this template: [hip template](hip-0000-template.md)
```
[the remark this answers is not in the document]
Comment (Nana-EC) [reply README.md:8]: Scratch that, I found this - https://github.com/hashgraph/hedera-improvement-proposal/pull/1037/files#diff-3a9ff98c9accc67b55c1e443d50986965c556688f49258e46d220891abcdfae0R4

[the remark this answers is not in the document]
Comment (jsync-swirlds) [reply README.md:8]: For additional context:

I've been soft recommending 80 characters for compatibility with github reviews.  The rendered documents wrap to window size, whatever that is, but github locks the review window to 80 characters in some cases, and 100 in a few others, no matter how wide the screen.

Given the primary interaction with the "code" form of these files is github code review; I believe 80 characters is a reasonable recommendation, provided we do not try to enforce it as a requirement.

In the **protocol buffers** I've recommended guidelines (in HIP-1037, as noted) that are more stringent, mostly to ensure readability in github, compatibility with the widest array of Javadoc (and Javadoc-like) processors (after compilation to various languages), and general simplicity.

File: assets/hip-0000-template/sample.proto [hunk 2]
```diff
@@ -0,0 +1,117 @@
+/**
+ * # Token Airdrop
```
[the remark this answers is not in the document]
Comment (jsync-swirlds) [reply assets/hip-0000-template/sample.proto]: Thanks, updated to make the sample status completely clear.

**Review by rbarkerSL — APPROVED** [review]

**Review by mgarbs — APPROVED** [review]

**Pull request merged**